test/util.js

```diff
@@ -216,7 +216,7 @@ describe('utility methods', () => {
 		it('should default to gutil.log', () => {
```

Comment:
shouldn't the test message be `it('should default to fancyLog, () => {`

Reply:
the jsDoc should also be updated where `[writable=gulp-util.log]` is specified
- https://github.com/mjeanroy/gulp-eslint/blob/refactor-remove-deprecated-gulp-util-dependency/index.js#L167
- https://github.com/mjeanroy/gulp-eslint/blob/refactor-remove-deprecated-gulp-util-dependency/index.js#L181

Reply:
Yes, what @ninbryan said. Good catch.

Reply:
@ninbryan Good catch, thanks!